Form: describe Smart Paste API (#8011)
DevExpress/devextreme-documentation: api-reference/10 UI Components/dxForm/1 Configuration/onSmartPasted.md

---
---
##### shortDescription
A function that is executed after an AI service populates the Form.

##### param(e): ui/form:SmartPastedEvent
Information about the event that caused the function's execution.

##### field(e.aiResult): Object

> Smart Paste results. This data populates the Form fields.

Reviewer: ```suggestion
Smart Paste results. Data from this object populates the Form.
```

##### field(e.component): {WidgetName}
UI component's instance.

##### field(e.element): DxElement
#include common-ref-elementparam with { element: "UI component" }

---

> The following code snippet displays a notification with Smart Paste operation result:

Reviewer 2: ```suggestion
The following code snippet displays a notification with the Smart Paste operation's result:
```

> The following code snippet displays a notification that indicates whether Smart Paste is successful:

Reviewer: "displays a notification that indicates" - all these words are pretty similar. You can display, notify, indicate... So I want to get rid of at least one verb there. 

```suggestion
The following code snippet displays a notification with Smart Paste operation result:
```
